### PR TITLE
Reorganized file tree and added parameter table to Dark case

### PR DESCRIPTION
**Pull request by rekomodo**
Got around to the data frames table and moved two MES cases that were out place. Also added exports to .gitignore.

### CODE REVIEW

File: report/_toc.yml [hunk 1]
```diff
@@ -13,13 +13,13 @@ parts:
...
         sections:
-        - file: verification/depleting_source
+        - file: verification/mes/depleting_source
         - file: verification/mms/simple
         - file: verification/mms/fluxes
         - file: verification/mms/discontinuity
         - file: verification/mms/trapping
         - file: verification/mms/heat_transfer
-        - file: verification/mes_radioactive_decay
+        - file: verification/mes/mes_radioactive_decay
```
Comment (RemDelaporteMathurin) [inline report/_toc.yml:22]:
```suggestion
        - file: verification/mes/radioactive_decay
```

mes is redundant here

**Comment by rekomodo** [comment]
think this is ready to go

**Comment by RemDelaporteMathurin** [comment]
The only issue is that it’s not straightforward that the header row is in units dpa

**Comment by rekomodo** [comment]
I agree. Should we just add dpa to every header's name for now?

**Comment by RemDelaporteMathurin** [comment]
Yep sounds good

File: report/validation/thermodesorption_spectra/dark/dark_tds.md [hunk 2]
```diff
@@ -284,3 +285,23 @@ plt.show()
...
+### Damage-induced trap parameters
+
+This table displays the neutron-induced traps' detrapping energy $E_p$ and their density per dpa dose.
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+dpa_no_zero = dpa_n_i | {}
+dpa_no_zero.pop(0)
+data = {"E_p" : detrapping_energies} | dpa_no_zero
```
Comment (RemDelaporteMathurin) [inline report/validation/thermodesorption_spectra/dark/dark_tds.md:300]: Similarly we need the units for E_p too

```suggestion
data = {"E_p (eV)" : detrapping_energies} | dpa_no_zero
```

**Comment by RemDelaporteMathurin** [comment]
The densities numbers don't have units (m-3). We could write it in the text

**Pull request merged**